packages/server/src/handlers/OptionsHandler.ts

```diff
@@ -7,8 +7,13 @@ import type http from 'node:http'
 // the Tus-Version header. It MAY include the Tus-Extension and Tus-Max-Size headers.
 export class OptionsHandler extends BaseHandler {
   async send(_: http.IncomingMessage, res: http.ServerResponse) {
+    const customerHeaders = this.options.optionsHeaders?.join(', ')
```

Comment:
can't use concat here, since typescript is complaining because the array is exported `as cont`.

We can either use the current approach of spreading into a new array as in this pr, HEADER as string[]